src/location.nit

```diff
@@ -55,6 +55,8 @@ class Location
 	var file: nullable SourceFile
 
 	# The starting line number (starting from 1)
+	#
+	# If `line_start==0` then the whole file is considered
```

Comment:
`line_start == 0`, with the spaces at the right place, the sentence is more elegant and easier to read.